sensors:
  logical_camera_1:
    type: logical_camera
    pose:
      xyz: [-0.639, -1.915, 1.003]
      rpy: [1.565, 0.937, 0.006]
  logical_camera_2:
    type: logical_camera
    pose:
      xyz: [-0.639, -1.155, 1.003]
      rpy: [1.565, 0.937, 0.006]
  logical_camera_3:
    type: logical_camera
    pose:
      xyz: [-0.639, -0.385, 1.003]
      rpy: [1.565, 0.937, 0.006]
  logical_camera_4:
    type: logical_camera
    pose:
      xyz: [-0.639, 0.407, 1.003]
      rpy: [1.565, 0.937, 0.006]
  logical_camera_5:
    type: logical_camera
    pose:
      xyz: [-0.639, 1.142, 1.003]
      rpy: [1.565, 0.937, 0.006]
  logical_camera_6:
    type: logical_camera
    pose:
      xyz: [-0.639, 1.910, 1.003]
      rpy: [1.565, 0.937, 0.006]
  logical_camera_7:
    type: logical_camera
    pose:
      xyz: [1.383, 3.35, 1.308]
      rpy: [-3.141, 1.20, -3.11]
  break_beam_1:
    type: break_beam
    pose:
      xyz: [1.6, 1.31, 0.91] 
      rpy: [0, 0, 'pi']
  break_beam_2:
    type: break_beam
    pose:
      xyz: [1.6, -0.63, 0.91] 
      rpy: [0, 0, 'pi']
  laser_profiler_1:
    type: laser_profiler
    pose:
      xyz: [1.21, 4, 1.6]
      rpy: ['-pi', 'pi/2', 'pi/2']
